src/main/kotlin/com/terraformation/backend/support/atlassian/model/ServiceRequestFieldsModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.terraformation.backend.support.atlassian.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
These model classes could use one-line KDoc comments to say what they're for. At first glance, it's not obvious what the difference between `ServiceRequestFieldsModel` and `ServiceRequestTypeModel` is other than that one has a name instead of a summary and one has an ID.